### ISSUE TITLE
Index DataFrame column using `['column_name']` syntax will allow for numeric column names.

### ISSUE THREAD

**Issue by StevenCHowell** [bokeh/datashader#284]
As demonstrated in the example from [this stack overflow response](http://stackoverflow.com/a/42164577/3585557), when converting a NumPy array into a Dataframe, the column labels are initially numeric.  When trying to index by these column labels, I realized the backend DataFrame indexing is using the attribute method, `df.column_label`, and not the `df[column_label]` method.  The current method supports less variation in column label in that it only supports valid python identifiers, e.g., column labels cannot be purely numeric.  [This Pandas' doc page](http://pandas.pydata.org/pandas-docs/stable/indexing.html) has more details.  If possible, changing to the `[ ]` method (or `__getitem__ method`) would allow for a wider range of inputs, thereby simplifying the user experience.

**Comment by jbednar** [comment]
We should be able to fix this; doesn't seem too difficult.  Thanks for the suggestion!

**Comment by jbednar** [comment]
[A PR would hurry this process along, by the way! :-)]

**Comment by StevenCHowell** [comment]
I'll see what I can do.  I did not find any "contributing guide".  Is there more than just cloning the repo, branching, then doing a pull request?  Is the `setup.py` designed to accept the `develop` option?

**Comment by jbednar** [comment]
Right; that's all there is to it.  If you have already done ``conda install datashader``, it's probably easiest to do ``conda remove datashader`` (to remove datashader while leaving all the dependencies satisfied), and then do ``python setup.py develop``.  That's what I do, anyway!

**Comment by StevenCHowell** [comment]
Yeah, I got a minute to create a new environment and get it setup; a quick test head positive results. I should have time to tinker on this tomorrow.

**Comment by StevenCHowell** [comment]
I tested the code as it is (without the change made by pr #285), and it did take column names that would not work with the `df.col_name` syntax, e.g., `df['x values']`.  It seems the real issue was it did not take numeric column names.
